examples/libaflfuzzer.c

```diff
@@ -803,12 +803,7 @@ bool broker_message_hook(llmp_broker_t *broker, llmp_broker_clientdata_t *client
 
 int main(int argc, char **argv) {
 
-  if (argc < 4) {
-
-    SAYF("Usage: %s number_of_threads /path/to/input/dir /path/to/queue/dir", argv[0]);
-    exit(0);
-
-  }
+  if (argc < 4) { FATAL("Usage: %s number_of_threads /path/to/input/dir /path/to/queue/dir", argv[0]); }
```

Comment:
nonono :) you are breaking fuzzbench with that.

Reply:
How so?

Reply:
Ohh. Didn't know. Will revert it. 😅

Reply:
Reverted this

Reply:
> How so?

build script tests the generated binary, does not supply that many arguments and a return code != 0 -> fail .... :)